values/service-auth-provider-api.yaml.gotmpl

```diff
@@ -35,6 +35,8 @@ app:
         value: {{ requiredEnv "S2S_SECRET_TASK_CONFIGURATION_API" }}
       - key: "MICROSERVICEKEYS_WA_CASE_EVENT_HANDLER"
         value: {{ requiredEnv "S2S_SECRET_CASE_EVENT_HANDLER" }}
+      - key: "MICROSERVICEKEYS_CAMUNDA_BPM"
+        value: {{ requiredEnv "S2S_SECRET_CAMUNDA_BPM" }}
```

Comment:
does this mean that s2s for camunda was not configured correctly?

Reply:
This is used when generating a s2s token rather than when verifying, previously we were just verifying but now camunda-bpm also generates them to be able to talk to the task-configuration service therefore the secret is needed.

Reply:
Yes,  I understood when I saw the other PRs. Thanks